node-js-basics-101/node_modules/xmlhttprequest-ssl/tests/test-events.js

```diff
@@ -0,0 +1,50 @@
+var sys = require("util")
+  , assert = require("assert")
+  , http = require("http")
+  , XMLHttpRequest = require("../lib/XMLHttpRequest").XMLHttpRequest
```

Comment:
__[jshint]__
* Redefinition of 'XMLHttpRequest'.
* Misleading line break before ','; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->